Content.Server/GuideGenerator/ChemistryJsonGenerator.cs

```diff
@@ -38,6 +38,7 @@ public static void PublishJson(StreamWriter file)
         var serializeOptions = new JsonSerializerOptions
         {
             WriteIndented = true,
+            NumberHandling = System.Text.Json.Serialization.JsonNumberHandling.AllowNamedFloatingPointLiterals, // Corvax-fix
```

Comment:
Используй один контекст для PR'а
```suggestion
            NumberHandling = System.Text.Json.Serialization.JsonNumberHandling.AllowNamedFloatingPointLiterals, // Corvax-Wiki
```